raiden/ui/cli.py

```diff
@@ -204,6 +205,13 @@ def options(func):
             help="Show all configuration values used to control Raiden's behavior",
             is_flag=True,
         ),
+        option(
+            "--blockchain-sync-interval",
+            help="Time interval after which to check for new blocks (in seconds)",
+            default=DEFAULT_BLOCKCHAIN_SYNC_INTERVAL,
+            show_default=True,
+            type=click.FloatRange(min=0.1, max=10.0),
```

Comment:
Why these numbers? I mean they are fine for now but if somehow we ended up watching a chain with blocktimes bigger than 20seconds (keep in mind Bitcoin is 10 mins) these limits would make no sense.

Reply:
I wanted to set some limits, but the numbers here are arbitrary of course.

Reply:
now that you removed range is it possible to pass a negative number? (not sure)

Reply:
It will probably be possible to pass it, and cause a crash when starting the AlarmTask. Maybe we keep a lower limit...

Reply:
yeah good point @BOR4 and @jomuel , a lower limit here is definitely sensible.

Reply:
I changed it back to `0.1`.